src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -400,28 +428,32 @@ const getMarketFields = (fieldSchema: FieldSchema, offerQuote: OfferQuote) => {
     offerQuote,
   )
   return {
-    fields: isSwedishAppartment
+    marketFields: isSwedishAppartment
       ? (fieldSchema as SwedishApartmentFieldSchema).swedishApartment
       : (fieldSchema as SwedishHouseFieldSchema).swedishHouse,
     key: isSwedishAppartment ? 'swedishApartment' : 'swedishHouse',
   }
 }
 
+const validateFields = (fields: MarketFields | CommonFieldSchema) =>
+  Object.entries(fields).reduce(
+    (acc, t) => ({
+      ...acc,
+      ...getValidationSchemaHelper(t),
```

Comment:
Not you that wrote this code, but what is `t` in this context 😄

Reply:
haha yes indeed what does t mean, good catch, I can  rename that one!